### PR TITLE
kubenurse: fix ingress

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
* configure deletion of ingress and service when disabled
* use exact path for ingress check

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

File: cluster/manifests/kubenurse/service.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{{ if eq .Cluster.ConfigItems.network_monitoring_enabled "true" }}
-  {{ if or (eq .Cluster.ConfigItems.network_monitoring_check_kubenurse_service "true") (eq .Cluster.ConfigItems.network_monitoring_check_kubenurse_ingress "true")}}
+# {{ if eq .Cluster.ConfigItems.network_monitoring_enabled "true" }}
+# {{ if or (eq .Cluster.ConfigItems.network_monitoring_check_kubenurse_service "true") (eq .Cluster.ConfigItems.network_monitoring_check_kubenurse_ingress "true")}}
```
Comment (linki) [inline cluster/manifests/kubenurse/service.yaml:2]: Why are the `ifs` commented out?

Comment (mikkeloscar) [reply cluster/manifests/kubenurse/service.yaml:2]: This is to make the file valid yaml. 😅

Comment (AlexanderYastrebov) [reply cluster/manifests/kubenurse/service.yaml:2]: This allows auto-formatting and syntax check by IDE

**Comment by AlexanderYastrebov** [comment]
:+1:

File: cluster/manifests/kubenurse/service.yaml (hunk 1 above)
Comment (linki) [reply cluster/manifests/kubenurse/service.yaml:2]: Ohhh, smart! 🤗

**Pull request merged**